concent_api/concent_api/system_check.py

```diff
@@ -0,0 +1,74 @@
+from django.core.checks     import Error, register
```

Comment:
So as we agreed in previous review, lets at least try to keep alphabetic order, and make multi line imports, like this:
```
from from django.core.checks     import (
    Error, 
    register,
)
```

When importing  this way, please add , to last object, so next person will not need to modify this line.

Reply:
OK, but let's stick to the style I suggested in the other pull request:

``` python
from django.core.checks import Error
from django.core.checks import register
```

Reply:
:+1:

Reply:
Done